src/consul.py

```diff
@@ -25,7 +24,10 @@ def restore_kv(self, restore_file):
 
         for line in restore_file:
             kv = line.split(':')
-            transaction_data.append({ 'KV' : { 'Verb': 'set', 'Key': kv[0], 'Value': kv[1] } })
+            if kv[1] == 'None\n':
+                logging.warn(f'Not restoring key {kv[0]}, value is "None"')
```

Comment:
Observed that locks like `consul-alerts/leader` and `replicator/leader` have `None` value in backup files that we should be aware of.